README.md

```diff
@@ -193,3 +196,8 @@ In System Preferences > Security & Privacy > Privacy > Accessibility, you should
 % rake yum:build
 % rake msi:build
 ```
+
+### Note for bump up the package version
+
+It assumes that Gemfile is used with Bundler's multiplatform feature,
+so bundler must be 2.2.0.rc1 or later.
```

Comment:
If Gemfile should be subject here and avoid to use passive voice:

```diff
-is used
+works
```

is correct what you mean?

---

It assumes that Gemfile *works* with Bundler's multiplatform feature,
so bundler must be 2.2.0.rc1 or later?

Reply:
Ah, It should be. I'll fix it.

Reply:
I've rebased with master.